src/main/scala/uk/gov/dhsc/htbhf/ClaimSimulation.scala

```diff
@@ -36,6 +36,13 @@ class ClaimSimulation extends Simulation {
     .exec(http("send_dob")
       .post("/enter-dob").formParam("dob-day", "1").formParam("dob-month", "11").formParam("dob-year", "1980").formParam("_csrf", "${csrf_token}"))
 
+    .exec(http("are_you_pregnant")
+      .get("/are-you-pregnant")
+      .check(status.is(200))
+    )
+    .exec(http("send_are_you_pregnant")
```

Comment:
Do you not need to set the value of "Are you pregnant?" to either true or false?

Reply:
No, we're not sending yes or no. Just the due date which is in the next step.